### DISCUSSION TITLE
Add Battery SOC (%) to Energy Dashboard

### DISCUSSION THREAD

**Discussion by RPI-PMK** [home-assistant/frontend#12603] [Other feature requests]
Adding the battery SOC (%) to dash board would help understand what the whole system is doing in single place.

Could also add a line plot onto energy plot to give a clearer indication of what energy is available in battery. This can then be read with the energy sources to see reasons for where energy is coming from.

Hope this makes sense.......

**Comment by jaal2001** [comment]
Would be great to have that available. As the SOC is shown (at least for my FoxESS) already in the device overview, maybe it is easy to add that to the Energy Dashboard?
<img width="490" alt="Bildschirmfoto 2024-03-15 um 12 14 42" src="https://github.com/home-assistant/frontend/assets/3871945/3eca09d2-dd6a-48da-8bf9-0cdf5fc0abc3">
